lib/pure/sugar.nim

```diff
@@ -148,6 +160,34 @@ macro `->`*(p, b: untyped): untyped =
 
   result = createProcType(p, b)
 
+macro `!=>`*(p, b: untyped): untyped =
+  ## Syntax sugar for anonymous `func` procedures. Supports pragmas.
+  since (1, 3):
```

Comment:
I'm not sure if it works, but this should be applied as a pragma to the macro. Otherwise if somebody asks for an older version they will get a confusing error (macro that fails). Also, I think you've got the version number wrong here.